### PR TITLE
Lpalac changing program

### PR DESCRIPTION
**Pull request by lPalac**
znan da su mi slova ne super kopirana veselim se vasim komentarima da cujem sto sam sve falija jer mislim da imam jos nekih malih gresaka

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <link href="https://fonts.googleapis.com/css?family=Merriweather:900|Roboto" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="./styles/stil.css">
+    <title>2.Domaci rad</title>
+</head>
+
+<body>
+
+    <section class="container">
+
+        <div id="tekst">
```
Comment (bvego) [inline index.html]: Avoid using ids for styling :)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            <h1>Život u Splitu</h1>
+            <p>Consulted perpetual of pronounce me delivered.
+                Too months nay end change relied who beauty wishes matter.
+                Shew of john real park so rest we on. Ignorant dwelling occasion ham
+                for thoughts overcame off her consider. Polite it elinor is depend.
+            </p>
+            <button class="botuni">Građani</button>
```
Comment (bvego) [inline index.html]: The class names should be in singular form most of the times, exceptions are mostly the containers of elements

File: styles/stil.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+background-color: lightgray;
+}
+#sekcija{
+    width: 625px;
+    margin: 25px 0px;
+
+}
+.container {
+    width: 625px;
+    margin: 25px auto;
```
Comment (bvego) [inline styles/stil.css]: 🙅‍♂️ 🛑 ⏹ 

The task said that spacing should not be done with margins or padding, what is the 3rd thing we can use? :)

File: styles/stil.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+#slika{
+    position: initial;
+    border-radius: 4px;
+    max-width: 625px;
+    width: 100%;
+
+}
+.botuni{
+    font-family: 'Merriweather', serif;
+    display: inline-block;
```
Comment (bvego) [inline styles/stil.css]: buttons are inline-block by default

File: index.html [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+    <title>2.Domaci rad</title>
+</head>
+
+<body>
+
+    <section class="container">
+
+        <div id="tekst">
+            <h1>Život u Splitu</h1>
+            <p>Consulted perpetual of pronounce me delivered.
```
Comment (bvego) [inline index.html]: These paragraphs should be a bit smaller in width

File: index.html (hunk 2 above)
Comment (bvego) [reply index.html]: Also, the buttons should be pushed down a bit more

**Comment by bvego** [comment]
Btw. boje nisu toliko bitne

File: styles/stil.css (hunk 3 above)
Comment (lPalac) [reply styles/stil.css]: border?

File: index.html (hunk 2 above)
Comment (lPalac) [reply index.html]: a sta treba bi ih pushat sa borderon ?

**Pull request closed without merging**